src/common/options.rs

```diff
@@ -611,13 +620,14 @@ pub enum Utterances {
 #[derive(Debug, PartialEq, Eq, Clone, Hash)]
 #[non_exhaustive]
 pub enum Multichannel {
-    #[allow(missing_docs)]
-    Disabled,
     #[allow(missing_docs)]
     Enabled {
         #[allow(missing_docs)]
         models: Option<Vec<Model>>,
```

Comment:
Having an optional value inside an enum suggests that maybe the enum should have an extra variant: one for `Enabled` and one for `Enabled with extra models`.  Something like:

```rust
pub enum Multichannel {
    /// Enable multichannel audio processing.
    ///
    /// Each audio channel will produce a separate transcript.
    Enabled,
    /// Disable multichannel processing.
    ///
    /// Audio channels will be demuxed to a single stream and produce a single ranscript.
    Disabled,
    /// Enable multichannel processing with a different model for each channel.  
    ///
    /// Models should be specified in the same order as the channels are encoded in the audio.
    /// If this option is used, do not specify a model with the [`OptionsBuilder::models`] method.
    ModelPerChannel { models: Vec<Model> },
}
```
Naming and documentation subject to bikeshedding.